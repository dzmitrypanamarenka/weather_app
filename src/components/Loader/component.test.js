import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import { Loader } from '../';

describe('Snapshot test',() => {
  it('should match snapshot', () => {
    const tree = renderer
      .create(<Loader />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe('Component render',() => {
  it('should shallow render Loader component', () => {
    const wrapper = shallow(<Loader />);
    expect(wrapper.find('.loader').length).toEqual(1);
  });
});
