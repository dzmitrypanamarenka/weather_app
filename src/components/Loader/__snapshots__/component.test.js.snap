// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test should match snapshot 1`] = `
<div
  className="loader"
/>
`;
